src/api/subscriptions.js

```diff
@@ -60,10 +60,10 @@ export class Subscriptions {
   clear() {}
 
   /**
-   * @param {?string=} encryptedDocumentKey
+   * @param {!GetEntitlementsParamsExternal=} params
    * @return {!Promise<!Entitlements>}
    */
-  getEntitlements(encryptedDocumentKey) {}
+  getEntitlements(params) {}
```

Comment:
Did you check that changing these params won't break anything? Check w/ @elijahsoria  to make sure

Reply:
Yes, I designed these changes to be backwards compatible. Elijah is double checking as well